libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -838,6 +852,29 @@ class MapboxNavigation @VisibleForTesting internal constructor(
             callback,
         )
     }
+    
+    @ExperimentalPreviewMapboxNavigationAPI
+    fun setPreviewRoutes(routes: List<NavigationRoute>) {
+        routesPreviewController.previewNavigationRoutes(routes)
+    }
+
+    @ExperimentalPreviewMapboxNavigationAPI
+    fun setPreviewPrimaryRoute(routes: NavigationRoute) {
```

Comment:
```suggestion
    @Throws(IllegalStateException::class)
    fun setPreviewPrimaryRoute(routes: NavigationRoute) {
```

I think we should advertise that this will fail if the `PreviewRoutes` has not been set